packages/oats-axios-adapter/index.ts

```diff
@@ -92,7 +92,7 @@ function createAxiosAdapter({
     const url = server + arg.path;
     const headers = removeUndefined({
       ...arg.headers,
-      ...(data instanceof FormData ? data.getHeaders() : {})
+      ...(data instanceof FormData && data.getHeaders ? data.getHeaders() : {})
```

Comment:
👏

Reply:
how do we set the correct content-type if we do not do it here?

Reply:
When form-data is used as payload, its Content-Type defaults to `multipart/form-data` in browser. In the console, i executed this line to check out the Request header. 
`FormData.prototype.getHeaders = function() {return {}}`
<img width="756" alt="image" src="https://github.com/smartlyio/oats/assets/122093405/3b51ee3c-d792-4ee9-859c-bb3e357128e6">